olmo/data/named_data_mixes.py

```diff
@@ -0,0 +1,1061 @@
+DATA_PATHS = {
+    "dolma17": [
```

Comment:
Maybe call this `dolma1_7`?

Reply:
I don't like that because it reads like `("dolma1", "7")`. Also, apparently all of our future releases are going to be named like "0727" anyways, so there is no way we'll get into confusion with a possible seventeenth version of Dolma.